golang/pkg/rnr/tasks.go

```diff
@@ -4,13 +4,15 @@ import (
 	"github.com/mplzik/rnr/golang/pkg/pb"
 )
 
+type TaskState int
+
 const (
-	PENDING = 0
-	RUNNING = 1
-	DONE    = 2
+	PENDING TaskState = iota
+	RUNNING
+	DONE
 )
 
-func taskSchedState(pbt *pb.Task) int {
+func taskSchedState(pbt *pb.Task) TaskState {
```

Comment:
Neat; I did not realize the consts in golang can be typed as enum this way :-)

Reply:
They are not really enums as in C/C# but rather a compile time trick, so treat them with care 😸

Reply:
Yup, but still -- marking them with a sort-of type is quite neat.